src/SFA.DAS.ReleaseTool.Core/Configuration/ReleaseConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace SFA.DAS.ReleaseTool.Core.Configuration
+{
+    public static class ReleaseConstants
+    {
+        public const string ReleaseName = "das-whitelist";
+        public const string IpAddressKey = "IPAddress";
```

Comment:
These might be better suited to some form of dictionary so it is easier to extend in the future